python/arcticdb/version_store/_normalization.py

```diff
@@ -317,16 +373,16 @@ def _normalize_single_index(index, index_names, index_norm, dynamic_strings=None
                 index_norm.fake_field_pos.append(0)
             log.debug("Index has no name, defaulting to 'index'")
         if isinstance(index, DatetimeIndex) and index.tz is not None:
-            index_tz = get_timezone(index.tz)
+            index_tz = index
```

Comment:
I think the naming has gone a bit weird here, `index_tz` being `pd.Index or np.ndarray` is odd